views/layout.hbs

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>IronBeers</title>
+    <link rel="stylesheet" href="/stylesheets/styles.css" />
+  </head>
```

Comment:
Here, you imported a Bootstrap `navbar`, but you forgot to link your layout to it's stylesheet.
```
<link
      rel='stylesheet'
      href='https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css'
      integrity='sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T'
      crossorigin='anonymous'
    />
```